packages/aragon-wrapper/src/rpc/handlers/events.js

```diff
@@ -1,8 +1,7 @@
 export default function (request, proxy) {
-  const fromBlock = request.params && request.params[0]
```

Comment:
Ahh shoot, we need to be backwards compatible here with `@aragon/api@1`, as old apps will still be using the first parameter as the `fromBlock`.

Reply:
I thought this PR was part of a major release, which'll break API

Reply:
Unfortunately `@aragon/wrapper` will have to stay compatible with older versions of `@aragon/api`, since it acts as the "server" :(

Reply:
So essentially all DAOs, including ones with non upgraded apps using the old API will be affected by an upgrade to `@aragon/wrapper` because it's imported by `aragon/aragon` and `apm-serve`: 

https://github.com/aragon/apm-serve/blob/b7dfacd084f60b1fa9e1cf01aa459004d3f06d54/lib/apm-router.js#L39-L41

will always load the latest?

Reply:
Yes.